### ISSUE TITLE
Class keyword arguments are not supported

### ISSUE THREAD

**Issue by shamrin** [microsoft/pyright#133]
**Describe the bug**
Pyright does not understand class keyword arguments.

**To Reproduce**
Try to use `mypy_extensions.TypedDict` with `total=False` argument from [mypy documentation][1]:

```python
from mypy_extensions import TypedDict

class Translations(TypedDict, total=False):
    de: str
    en: str
```

[1]: https://mypy.readthedocs.io/en/latest/more_types.html#mixing-required-and-non-required-items

**Expected behavior**
I expect Pyright not to be confused by `total=False`.

**Screenshot**
<img width="609" alt="image" src="https://user-images.githubusercontent.com/510678/58157933-0dc33380-7c82-11e9-8da7-5fe3c5d72535.png">

**VS Code extension**
Name: Pyright
Id: ms-pyright.pyright
Description: VS Code static type checking for Python
Version: 1.0.31
Publisher: ms-pyright
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=ms-pyright.pyright

**Comment by erictraut** [comment]
Thanks for the bug report. I wasn't aware that Python supported this, but indeed it does. I just found the statement in the python spec:
```
3.3.3.1: Any other keyword arguments that are specified in the class definition are passed through to all metaclass operations described below.
```
This is an easy fix - mostly just involves removing some existing code. I'll have it fixed shortly.

**Comment by erictraut** [comment]
This is now fixed in 1.0.32, which I just published. Thanks again for the bug report.
